interface/usergroup/usergroup_admin_add.php

```diff
@@ -7,7 +7,7 @@
  * @link      http://www.open-emr.org
  * @author    Brady Miller <brady.g.miller@gmail.com>
  * @author    Rod Roark <rod@sunsetsystems.com>
- * @copyright Copyright (c) 2017-2018 Brady Miller <brady.g.miller@gmail.com>
+ * @copyright Copyright (c7) 2017-2018 Brady Miller
```

Comment:
Why are you changing the copyright notice here?  You should feel free to add your own copyright after this, but don't modify other people's copyright notices.